src/components/MakeWidget/subcomponents/RateField/RateField.tsx

```diff
@@ -33,8 +33,13 @@ export const RateField: React.FC<RateFieldProps> = ({
     setCurrentRate(rate);
   }, [rate]);
 
+  useEffect(() => {
+    setInvertPair(false);
+  }, [token1, token2, rate]);
```

Comment:
```suggestion
  }, [token1, token2]);
```

When changing amounts dont invert pair.